workspace/modeling.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
```

Comment:
This is a great change!! But try this...
# title

```
code code code
```
[links](https://github.com/dblodgett-usgs/si_docker_jupyter_training/)

<img width="447" alt="Screen Shot 2019-06-10 at 6 30 45 AM" src="https://user-images.githubusercontent.com/1492803/59721633-5c171400-91e7-11e9-8ebd-bf79f11d9a5e.png">